lib/spack/spack/cmd/compiler.py

```diff
@@ -84,8 +83,6 @@ def compiler_find(args):
 
     """
     paths = args.add_paths
-    if not paths:
-        paths = get_path('PATH')
```

Comment:
I'm not sure about this change. When you run `spack compiler find`, it should search your `PATH` for any available compilers. Isn't this important?